sql/maintenance_utils.sql

```diff
@@ -39,11 +39,24 @@ CREATE OR REPLACE FUNCTION @extschema@.compress_chunk(
     hypercore_use_access_method BOOL = NULL
 ) RETURNS REGCLASS AS '@MODULE_PATHNAME@', 'ts_compress_chunk' LANGUAGE C VOLATILE;
 
+-- Alias for compress_chunk above.
+CREATE OR REPLACE PROCEDURE @extschema@.convert_to_columnstore(
+    uncompressed_chunk REGCLASS,
+    if_not_compressed BOOLEAN = true,
```

Comment:
Does the if_not_compressed make sense anymore. I guess it actually checks whether it is already a compressed chunk, not whether it is actually compressed.

```suggestion
    if_not_columnarstore BOOLEAN = true,
```